src/site/content/en/blog/codelab-sign-in-form-best-practices/index.md

```diff
@@ -36,28 +36,28 @@ accessibility, and add meaning to your markup.
 
 2. Add the following code to the `<body>` element:
 
-```html
-<form action="#" method="post">
-  <h1>Sign in</h1>
-  <section>
-    <label>Email</label>
-    <input>
-  </section>
-  <section>
-    <label>Password</label>
-    <input>
-  </section>
-  <button>Sign in</button>
-</form>
-```
-
-Here's how your `index.html` file should look at this point:
-
-{% Glitch {
-  id: 'sign-in-form-codelab-1',
-  path: 'index.html',
-  height: 480
-} %}
+  ```html
+  <form action="#" method="post">
+    <h1>Sign in</h1>
+    <section>
+      <label>Email</label>
+      <input>
+    </section>
+    <section>
+      <label>Password</label>
+      <input>
+    </section>
+    <button>Sign in</button>
+  </form>
+  ```
+
+   Here's how your `index.html` file should look at this point:
+
+  {% Glitch {
+    id: 'sign-in-form-codelab-1',
+    path: 'index.html',
+    height: 480
+  } %}
```

Comment:
The numbering kept resetting after each code block. I looked into this and it seems that we need to indent the code blocks for Markdown to realize that the code blocks are part of the previous list item. I made this change here and throughout the codelab.